src/main/java/com/book/controller/AuthorizationController.java

```diff
@@ -0,0 +1,47 @@
+package com.book.controller;
+
+import com.book.model.JwtModel;
+import com.book.model.UsersEntity;
+import com.book.service.MyUserDetailsService;
+import com.book.util.JwtUtils;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.http.ResponseEntity;
+import org.springframework.security.authentication.AuthenticationManager;
+import org.springframework.security.authentication.BadCredentialsException;
+import org.springframework.security.authentication.UsernamePasswordAuthenticationToken;
+import org.springframework.security.core.userdetails.UserDetails;
+import org.springframework.web.bind.annotation.PostMapping;
+import org.springframework.web.bind.annotation.RequestBody;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RestController;
+
+@RestController
+@RequestMapping("/login")
+public class AuthorizationController {
+    AuthenticationManager authenticationManager;
+    JwtUtils jwtUtils;
+    MyUserDetailsService myUserDetailsService;
+
+    @Autowired
```

Comment:
there's a trick with Lombok+Contructor injection that can make your code cleaner:
https://www.baeldung.com/spring-injection-lombok